downstream/modules/platform/con-controller-workflow-states.adoc

```diff
@@ -0,0 +1,14 @@
+[id="controller-workflow-states"]
+
+= Workflow states
+
+The workflow job can have the following states (no Failed state):
+
+* Waiting
+* Running
+* Success (finished)
+* Cancel
+* Error
+* Failed
```

Comment:
Why is Failed here if there's no Failed state?

Reply:
I thought this was odd too, I need to ask an SME about this.